### PR TITLE
[docs] migration pattern docs update

### PR DESCRIPTION
**Pull request by snopoke**
Small tweaks to the migration pattern docs.

### Safety story
Docs only

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by millerdev — APPROVED** [review]
Nit: most of this document is wrapped at 80 characters (with at least one exception being the old line 256, which was not wrapped at all). The new sections you added/edited appear to be wrapped inconsistently somewhere between 80 and 115 characters. Consider matching the style of the surrounding code. VS Code has a "Rewrap/Unwrap Text at Column" command that makes this easy; I'm guessing most modern editors have something similar.

**Comment by snopoke** [comment]
> Nit: most of this document is wrapped at 80 characters (with at least one exception being the old line 256, which was not wrapped at all). The new sections you added/edited appear to be wrapped inconsistently somewhere between 80 and 115 characters. Consider matching the style of the surrounding code. VS Code has a "Rewrap/Unwrap Text at Column" command that makes this easy; I'm guessing most modern editors have something similar.

This was harder than it sounds and I ended up doing it manually. The file is quite inconsistent so wrapping to 80 changed most lines (shrinking some and growing others). I also didn't find any tools that respected the heading formats. I'd be curious to see how well your VS Code tool works.

File: docs/migrations_in_practice.rst [hunk 1]
```diff
@@ -269,6 +269,7 @@ and it's a good idea that anything run on a production environment is on the mas
 Run your schema migration and management command directly:
 
     ``cchq <ENV> django-manage --release=<NAME> migrate <APP_NAME>``
+    ``cchq <ENV> django-manage --release=<NAME> my_data_migration_command``
```
Comment (esoergel) [inline docs/migrations_in_practice.rst:277]: :+1:

**Review by esoergel — COMMENTED** [review]
nice changes, thanks

**Pull request merged**